### ISSUE TITLE
Deprecation notice in ee/ExpressionEngine/Addons/stats/mod.stats.php, line 167

### ISSUE THREAD

**Issue by bnetz** [ExpressionEngine/ExpressionEngine#2505]
After updating to EE7 + PHP8 we get the following notice:
```
Deprecated
preg_replace(): Passing null to parameter #3 ($subject) of type array|string is deprecated
ee/ExpressionEngine/Addons/stats/mod.stats.php, line 167
Severity: E_DEPRECATED
```

**Environment Details:**
 - EE 7.2
 - PHP 8.1.12

**Comment by bnetz** [comment]
Problem still persists after upgrading to 7.2.1

**Comment by bnetz** [comment]
Solved with 7.2.2, thanks!

**Comment by matthewjohns0n** [comment]
Awesome, glad to hear it!
